File: A/C++/Processor.cpp
#include <string>
#include <iostream>

using namespace std;

// Kelas Processor
class Processor
{
    // Atribut Private
    private:
        string name;
        int price;

    // Atribut Public
    public:
        // Constructor
        Processor()
        {
        
        }

        // Constructor Input
        Processor(string name, int price)
        {
            this->name = name;     
            this->price = price;
        }

        // Metode Setter
        void setName(string name)
        {
            this->name = name;
        }

        void setPrice(int price)
        {
            this->price = price;
        }

        // Metode Getter
        string getName()
        {
            return this->name;
        }

        int getPrice()
        {
            return this->price;
        }

        // Metode Print
        void printProcessor()
        {
            cout << "===============================" << endl;
            cout << "Prosessor:" << endl;
            cout << "===============================" << endl;
            cout << "Nama           : " << this->name << endl;
            cout << "Harga          : " << "Rp" << this->price << endl;
        }

        // Destructor
        ~Processor()
        {   

        };
};

File: B/Java/Processor.java
// Kelas Processor
class Processor
{
    // Atribut Private
    private String name;
    private int price;

    // Constructor
    Processor()
    {
        
    }

    // Constructor Input
    public Processor(String name, int price)
    {
        this.name = name;
        this.price = price;
    }

    // Metode Getter dan Setter
    public String getName()
    {
        return name;
    }

    public void setName(String name)
    {
        this.name = name;
    }

    public int getPrice()
    {
        return price;
    }

    public void setPrice(int price)
    {
        this.price = price;
    }

    // Metode Print
    public void printProcessor()
    {
        System.out.println("==================================");
        System.out.println("Prosessor:");
        System.out.println("==================================");
        System.out.println("Nama            : " + this.name);
        System.out.println("Harga           : " + "Rp" + this.price);
    }
}
